packages/react-network/src/components/FontSource.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import {CspDirective} from '@shopify/network';
+import SetCspDirective from './SetCspDirective';
+
+interface Props {
+  sources: string | string[];
+}
+
+export default function FontSource({sources}: Props) {
+  return <SetCspDirective directive={CspDirective.FontSrc} value={sources} />;
```

Comment:
I don't see a ton of value in having a separate component for every CspDirective. Can we just do `<SetCspDirective directive={CspDirective.[...]Src}/>` where we need to set it? The `values` prop type appears to be the same for all of them.

Reply:
There is a component for this, but IMO it is much nicer to provider more tailored to components. Some directives, like the `FooSource` ones, accept arrays of strings. Some, like `UpgradeInsecureRequests`, accept a boolean. A single component means they don't get any tips on what to pass in.